init.coffee

```diff
@@ -13,11 +13,30 @@ exports.configure = [
     message: 'Do you need netlify hosting?'
   },
   {
+    type: 'list'
+    name: 'css'
+    message: 'What would you like to use as your CSS Preprocessor?'
+    choices: ['postcss', 'stylus']
```

Comment:
I'm actually using poststylus in share-button: https://github.com/carrot/share-button/blob/master/gulpfile.js#L24